linux/engine.mk

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Before editing these, make sure you have read the documentation on how to
+# build the flutter engine here:
+# https://github.com/flutter/engine/blob/master/CONTRIBUTING.md
+#
+# $(FLUTTER_ENGINE_ROOT) is the variable pointing to the git checkout of the
+# flutter engine. Here, the flutter engine path is relative to
+# `flutter-desktop-embedding/linux`
+#
+# This variable will be used when building `example_flutter/` to pass the
+# `--local-engine-src-path` flag to the flutter binary.
+FLUTTER_ENGINE_ROOT=../../engine
```

Comment:
We should probably give this a less generic name (flutter-engine?) to avoid conflicts and/or confusion if this is part of a larger set of repositories someone is using, and then update the engine-cloning instructions accordingly.

Reply:
Sounds like a good idea.  I'll update the README as well to reflect that.